sunflower-land/sunflower-land
# [CHORE] Combine hud buttons

# Description

- combine chest and buildings HUD buttons
  - the buildings button is rarely used.  Removing it would save some screen real estate especially for smaller screens
  - buildings tab will only show on your home island

![image](https://user-images.githubusercontent.com/107602352/216780298-cc457962-5484-4278-bbd2-6a49d6f4e0fd.png)

## Type of change


# How Has This Been Tested?

- navigate between the 3 tabs in inventory
- go to other islands / visit friends.  The 3rd tab should be hidden
- go through buildings tutorial
- place buildings

# Checklist:
